src/map.js

```diff
@@ -1963,10 +1963,14 @@ var map = function (arg) {
    * @param {number} opacity The opacity in the range [0, 1].
    * @param {object} elem A jQuery element that might have a transform.
    * @param {HTMLImageObject} img The image or canvas to draw to the canvas.
+   * @param {string} [mixBlendMode] the mix-blend-mode used to add this layer.
    * @private
    */
-  function drawLayerImageToContext(context, opacity, elem, img) {
+  function drawLayerImageToContext(context, opacity, elem, img, mixBlendMode) {
     context.globalAlpha = opacity;
+    if (mixBlendMode) {
```

Comment:
would it be not possible to just have the user pass any one type of globalCompositeOperation (source in/out, over etc?)

Reply:
Yes, any of the globalCompositeOperation values are supported.  They are called mix-blend-mode in css and globalCompositeOperation in canvas.